open-banking-accelerator/components/event-notifications/com.wso2.openbanking.accelerator.event.notifications.service/src/main/java/com/wso2/openbanking/accelerator/event/notifications/service/handler/DefaultEventSubscriptionServiceHandler.java

```diff
@@ -47,8 +47,8 @@ public void setEventSubscriptionService(EventSubscriptionService eventSubscripti
     /**
      * This method is used to create event subscriptions.
      *
-     * @param eventSubscriptionRequestDto
-     * @return EventSubscriptionResponse
+     * @param eventSubscriptionRequestDto Event Subscription DTO
```

Comment:
```suggestion
     * @param eventSubscriptionRequestDto - Event Subscription DTO
```

Reply:
Fixed in https://github.com/wso2/financial-open-banking/pull/3/commits/a31eef6176e5003bd359e183aae45a90a5874f2b